### PR TITLE
Add Overlapping Community Leaderboards to PSI Section

### PR DESCRIPTION
**Pull request by stevenelleman**
If there are shared communities AND top 10 leaderboard entries: 
Screenshot 2024-11-06 at 12 17 12 AM

If there are no shared communities OR no top 10 leaderboard entries: 
Screenshot 2024-11-06 at 12 16 15 AM

### CODE REVIEW

File: apps/frontend/src/pages/people/[username].tsx
Comment (stevenelleman) [inline apps/frontend/src/pages/people/[username].tsx:210]: *distance running over last month

File: apps/frontend/src/pages/profile/overview.tsx
Comment (stevenelleman) [inline apps/frontend/src/pages/profile/overview.tsx:66]: In one location we had username at the top and display below it, and two cases with the reverse. Given that quite a few users have not filled out the display name, and username is required, I swapped the naming to consistently put username first and display name below it.

File: packages/types/src/chip/index.ts
Comment (stevenelleman) [inline packages/types/src/chip/index.ts:131]: Going to sleep on this, whether it should be shared or not.

File: apps/frontend/src/pages/people/[username].tsx
Comment (AndrewCLu) [inline apps/frontend/src/pages/people/[username].tsx:248]: do we have devcon leaderboards set up?

**Review by AndrewCLu — APPROVED** [review]
approved pending testing with lanna + devcon leaderboards

File: apps/frontend/src/pages/people/[username].tsx
Comment (stevenelleman) [reply apps/frontend/src/pages/people/[username].tsx:246]: Shared between the two users, I'll change name to "SharedIssuerLeaderboards"

File: apps/frontend/src/pages/people/[username].tsx
Comment (RiverRuby) [inline apps/frontend/src/pages/people/[username].tsx:672]: I want to clean up this copy + styling

**Review by RiverRuby — CHANGES_REQUESTED** [review]
Don't merge until I get to update copy+styling later

**Comment by RiverRuby** [comment]
Closing since this is outdated and unclear that we want this for now. Discovering overlapping communities won't be useful until we have many more communities.

**Comment by stevenelleman** [comment]
> Closing since this is outdated and unclear that we want this for now. Discovering overlapping communities won't be useful until we have many more communities.

@RiverRuby the point of this was to surface leaderboard on tap (ie if you're top 10 on github contributions, it's surfaced), but if you do that you need to handle the case where you have multiple communities. The multiple community aspect is secondary.

**Pull request closed without merging**
